tests/test_data.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for the mibipret.data module."""
+
+# importing module
+path_package = '/home/alraune/GitHub/MiBiPreT/mibipret'
```

Comment:
This should not be necessary if you installed the package according to the documentation (with pip install), this is not necessary. The package will be added to the path automatically.

Reply:
Thanks a lot for the super fast review and the useful comments. I integrated them all. Surely, there is many more to include to the module and improvements of codes and test, but it is a good start. :-)

Reply:
Adapted.